### DISCUSSION TITLE
Issue with Global recording feature - pressing 'R' does not stop the recording.

### DISCUSSION THREAD

**Discussion by lioncladowls** [ggml-org/whisper.cpp#2096] [Q&A]
I'm currently using the global recording tool on Mac version Sonoma 14.2, running on an Apple M3 Max. Initially, the tool starts recording as expected, but I encounter an issue where it doesn't stop recording when I press the 'R' key, despite the interface indicating that 'R' should stop the recording. I've tried various key combinations to stop it, but none seem to work and there appears to be no related setting available.

Could you please help me understand if this is a known issue or if I might be missing something? Any guidance or a potential fix would be greatly appreciated.

**Comment by ulatekh** [comment]
This sounds like a Macintosh question that has nothing to do with Whisper.
